src/Utils/services.js

```diff
@@ -0,0 +1,99 @@
+import axios from 'axios'
+
+export const addToNotes = async (note,token) => {
+    try {
+      const resp = await axios.post(
+        "/api/notes",
+        { note },
+        { headers: { authorization: token } }
+      );
+
+    } catch (err) {
+      console.log(err);
+    }
+};
+
+export const updateNote = async (note, token) => {
+    try {
+      const resp = await axios.post(
+        `/api/notes/${note._id}`,
+        { note },
+        { headers: { authorization: token } }
+      );
+
+    } catch (err) {
+      console.log(err);
+    }
+};
+
+
+export const deletefromAllNotes = async (note, token) => {
+    const { _id } = note;
+    try {
+        await axios.delete(`/api/notes/archives/${_id}`,{headers:{authorization: token}});
+    } catch (err) {
+        console.log(err)
+  }
+}
+
+// -----------------------------------------------------------------------------------------------------------------------------
```

Comment:
remove these lines and write down comments on what the below function returns or does.